Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -2077,6 +2079,57 @@ int CLuaPedDefs::SetPedAimTarget ( lua_State* luaVM )
 }
 
 
+int CLuaPedDefs::SetPedStat ( lua_State* luaVM )
+{
+    // Verify the argument
+    CClientPed* pPed = NULL;
+    unsigned short usStat = 0;
+    float fValue = 0;
+    CScriptArgReader argStream ( luaVM );
+    argStream.ReadUserData ( pPed );
+    argStream.ReadNumber ( usStat );
+    argStream.ReadNumber ( fValue );
+
+    if ( !argStream.HasErrors () )
+    {
+        // Valid ped?
+        if ( pPed )
```

Comment:
This line is not necessary as the error case is already handled by `CScriptArgReader::HasErrors`